### PR TITLE
read st_masterdata as rds file; allow for credit md file to be read

### PR DESCRIPTION
**Pull request by jacobvjk**
Closes ADO 1607 https://dev.azure.com/2DegreesInvesting/2DegreesInvesting/_workitems/edit/1607

This PR:
- reads the `stress_test_master_data_*.rds` files as `rds`, not `rda`
- these files have been changed to RDS in the predecessor https://github.com/2DegreesInvesting/r2dii.stress.test.data/pull/27
- additionally the utility function create_stressdata_masterdata_file_paths() now allows for reading st master data for loans as well, by calling `read_company_data(path = stresstest_masterdata_files$loans)`

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -109,8 +109,9 @@ create_stressdata_masterdata_file_paths <- function(data_prep_timestamp, twodii_
   path_parent <- path_dropbox_2dii("PortCheck", "00_Data", "07_AnalysisInputs", data_prep_timestamp)
 
   paths <- list(
-    "stress_test_masterdata_debt.rda",
-    "stress_test_masterdata_ownership.rda"
+    "stress_test_masterdata_debt.rds",
+    "stress_test_masterdata_ownership.rds",
+    "stress_test_masterdata_credit.rds"
```
Comment (MirijaHa) [inline R/utils.R:114]: Just wondering, the now available data are not yet to actually be read in when running the loanbook flow?

Comment (jacobvjk) [reply R/utils.R:114]: no, but definitely will be the next change required on the loan book side

Comment (jacobvjk) [reply R/utils.R:114]: https://dev.azure.com/2DegreesInvesting/2DegreesInvesting/_workitems/edit/879

**Review by MirijaHa — APPROVED** [review]
🥇

**Pull request merged**